README.md

```diff
@@ -98,269 +99,56 @@ Currently we supports lazy calculation mode to reduce copy of tensors during cal
-10/12 unittest_util_func             OK             0.05s
-11/12 unittest_databuffer_file       OK             0.12s
-12/12 unittest_nntrainer_modelfile   OK             2.22s
-
-Ok:                 12
-Expected Fail:      0
-Fail:               0
-Unexpected Pass:    0
-Skipped:            0
-Timeout:            0
-```
-
-if you want to run particular test
-
-```bash
-$ meson -C build test $(test name)
-```
-
-2. Sample app test
-NNTrainer provides extensive sample app running test.
-
-Meson build with `enable-app` set to true
-```bash
-$ echo $(pwd)
-(project root)
-
-$ meson build -Denable-app=true
-The Meson build system
-Version: 0.54.3
-...
-Configuring capi-nntrainer.pc using configuration
-Run-time dependency GTest found: YES (building self)
-Build targets in project: 17
-
-Found ninja-1.10.0.git.kitware.jobserver-1 at /home/jlee/.local/bin/ninja
-
-$ ninja -C build test
-...
- 1/21 app_classification             OK             3.59s
- 2/21 app_classification_func        OK             42.77s
- 3/21 app_knn                        OK             4.81s
- 4/21 app_logistic                   OK             14.11s
- 5/21 app_DeepQ                      OK             30.30s
- 6/21 app_training                   OK             38.36s
- 7/21 app_classification_capi_ini    OK             32.65s
- 8/21 app_classification_capi_file   OK             32.04s
- 9/21 app_classification_capi_func   OK             29.13s
- ...
-```
-
-if you want to run particular example only
-
-```bash
-$ meson -C build test $(test name) #app_classification_capi_func
-```
-
-### Running Examples
-
-1. [Training](https://github.com/nnstreamer/nntrainer/blob/master/Applications/Training/README.md)
-
-After build, run with following arguments
-Make sure to put last '/' for the resources directory.
-```bash
-$./path/to/example ./path/to/settings.ini ./path/to/resource/directory/
-```
-
-To run the 'Training', for example, do as follows.
-
-```bash
-$ pwd
-./nntrainer
-$ LD_LIBRARY_PATH=./build/nntrainer ./build/Applications/Training/jni/nntrainer_training ./Applications/Training/res/Training.ini ./Applications/Training/res/
-../../res/happy/happy1.bmp
-../../res/happy/happy2.bmp
-../../res/happy/happy3.bmp
-../../res/happy/happy4.bmp
-../../res/happy/happy5.bmp
-../../res/sad/sad1.bmp
-../../res/sad/sad2.bmp
-
-...
-
-```
+#### [KNN Example](https://github.com/nnstreamer/nntrainer/tree/main/Applications/KNN)
 
-## Open Source License
+A transfer learning example with for image classification using the Cifar 10 dataset. TFlite is used for feature extractor and compared with KNN.
 
-The nntrainer is an open source project released under the terms of the Apache License version 2.0.
+#### [Logistic Regression Example](https://github.com/nnstreamer/nntrainer/tree/main/Applications/LogisticRegression)
+
+A logistic regression example using NNTrainer.
+
+## [Getting Started](https://github.com/nnstreamer/nntrainer/blob/main/docs/getting-started.md)
 
-## Contributing
```

Comment:
how about not removing this section?

Reply:
Thank you for the notification, I updated this part recovering the section.